lib/iris/cube.py

```diff
@@ -620,7 +621,8 @@ class Cube(CFVariableMixin):
     def __init__(self, data, standard_name=None, long_name=None,
                  var_name=None, units=None, attributes=None,
                  cell_methods=None, dim_coords_and_dims=None,
-                 aux_coords_and_dims=None, aux_factories=None):
+                 aux_coords_and_dims=None, aux_factories=None,
+                 cell_measures_and_dims=None):
```

Comment:
@marqh `cell_measures_and_dims` requires an associated entry in the `Kwargs` section of the doc-string.